plugins/modules/mso_schema_site_external_epg.py

```diff
@@ -84,7 +97,8 @@
     schema: Schema 1
     template: Template 1
     external_epg: External EPG 1
-    l3out: L3out1
+    l3out:
```

Comment:
this should not be dict input anymore correct?

Reply:
yes changed